scio-bigquery/src/main/scala/com/spotify/scio/bigquery/package.scala

```diff
@@ -68,6 +68,11 @@ package object bigquery {
    */
   type description = com.spotify.scio.bigquery.types.description
 
+  /** Enhanced version of [[TableReference]]. */
+  implicit class RichTableReference(private val r: TableReference) extends AnyVal {
+    def asString: String = s"${r.getProjectId}:${r.getDatasetId}.${r.getTableId}"
```

Comment:
Call it `asTableSpec` to differentiate from `toString`? Also most Google code refers to the string as "TableSpec".